File: Cpp/Unit01/Exercise06/src/exercise06.cpp
#include<iostream>
#include<fstream>
#include<string>
#include<cmath>

using namespace std;

int main() {
    float f;

    // writing out the result
    ofstream outcsv;
    ofstream outtab;
    outcsv.open("../Unit01/Exercise06/Outputs/output.csv");
    outtab.open("../Unit01/Exercise06/Outputs/output.txt");
    outcsv << "Number, Square\n";
    outtab << "Number \t Square\n";
    for (int i = 1; i <= 25; i++) {
        f = (float) i;
        outcsv << to_string(i) + ", " + to_string(f) + "\n";
        outtab << to_string(i) + "\t" + to_string(f) + "\n";
    }
    outcsv.close();
    outtab.close();

    return 0;
}

File: Fortran90/Unit01/Exercise06/src/exercise06.f90
program exercise06
    implicit none
    integer :: i

    open(1, file="../Unit01/Exercise06/Outputs/output.csv")
    open(2, file="../Unit01/exercise06/Outputs/output.txt")
    write(1,*) "Number, Square"
    write(2,*) "Number", char(9), "Square"
    do i = 1, 25
        write(1,*) i, ", ", real(i)*real(i)
        write(2,*) i, char(9), real(i)*real(i)
    end do
    close(1)
    close(2)
    
end program exercise06